### DISCUSSION TITLE
Customise default location for snippets data

### DISCUSSION THREAD

**Discussion by idiston** [pieces-app/support#135] [Ideas]
I'd like to suggest a feature, whereby as part of the installation process, a user can specify the 'save' location of the Pieces database and associated files.

For example, I would like to specify a Dropbox folder specifically for Pieces so that it can be accessed from any of the devices that I use.

**Comment by Thundercraft5** [comment]
Definitely a must! I work with a portable dev workspace on a USB flash drive across multiple machines. I would maybe like to add that this should also be configurable via in-app settings (desktop app).
